packages/ember-runtime/lib/system/native_array.js

```diff
@@ -138,9 +138,10 @@ var A;
 
 if (Ember.EXTEND_PROTOTYPES === true || Ember.EXTEND_PROTOTYPES.Array) {
   NativeArray.apply(Array.prototype);
-  A = function (arr = []) { return arr; };
+  A = function (arr = []) { return arr || []; };
 } else {
   A = function (arr = []) {
+    if (arr === null) { arr = []; }
```

Comment:
@mixonic Ha. It turns out the old behavior was inconsistent too. https://github.com/emberjs/ember.js/commit/586d4c6ab0a82abee0df1f331121e9c99433770e#diff-7 I think I'll just use the truthiness check and add a code comment. Then see what others think.